modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/test/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorTest.java

```diff
@@ -569,6 +593,16 @@ protected Set<Locale> createAvailableLocales(Locale... locales) {
 		return DDMFormTestUtil.createAvailableLocales(locales);
 	}
 
+	private DDMFormField _createDDMFormField(
+		String name, String type, String reference) {
```

Comment:
@devpedrotavares please sort this alphabetically

Reply:
Done!